pkg/config/config.go

```diff
@@ -3,16 +3,18 @@ package config
 import (
 	"context"
 	"fmt"
+	"math"
```

Comment:
To avoid confusion of the receiver's config, do we need to rename `pkg/config/config.go` to something else like `pkg/notify/controller.go`

Reply:
Done